src/uiSelectDirective.js

```diff
@@ -331,8 +333,11 @@ uis.directive('uiSelect',
               return;
             }
 
-            // Hide the dropdown so there is no flicker until $timeout is done executing.
-            dropdown[0].style.opacity = 0;
+           // Hide the dropdown so there is no flicker until $timeout is done executing.
+           if ($select.search === '' && !opened) {
+              dropdown[0].style.opacity = 0;
+              opened = true;
```

Comment:
Not that i've found, since it's already been rendered once - AFAIK the opacity change was for the initial render only.